ruleguard/testdata/bench_target1.go

```diff
@@ -18,12 +18,11 @@ package main
 
 // Import declaration declares library packages referenced in this file.
 import (
-	"fmt"       // A package in the Go standard library.
-	"io/ioutil" // Implements some I/O utility functions.
```

Comment:
This is the only place that makes me uncomfortable.
This is a testdata, it should probably be ignored by linters.
Is it a global replace mistake or does golangci-lint trigger a warning for this file too?